s3_to_knack.py

```diff
@@ -68,8 +68,17 @@ def get_pks(fields, app_name):
 
 
 def is_equal(rec_current, rec_knack, keys):
-    tests = [rec_current[key] == rec_knack[key] for key in keys]
-    return all(tests)
+    for key in keys:
+        # Comparing comma separated lists regardless of order
+        if isinstance(rec_current[key], str) and isinstance(rec_knack[key], str):
+            if ",\n" in rec_current[key] and ",\n" in rec_knack[key]:
+                if set(rec_current[key].split(",\n")) != set(rec_knack[key].split(",\n")):
+                    return False
+            elif rec_current[key] != rec_knack[key]:
+                return False
+        elif rec_current[key] != rec_knack[key]:
+            return False
+    return True
```

Comment:
@Charlie-Henry this is looking good to me, but i wonder if you'd be better off moving this into a field-specific handler, like the ones we have [in here](https://github.com/cityofaustin/atd-finance-data/blob/ch-list-compare/config.py). that way you don't have to run all this string checking on every column for ever row.

Reply:
...and in that case the handler would sort the comma-separated list of items to make sure the order is always consistent